packages/storybook/stories/TextInput.stories.jsx

```diff
@@ -30,6 +50,12 @@ Default.args = {
   ...defaultArgs,
 };
 
+export const Internationalization = I18nTemplate.bind({});
```

Comment:
Just as an fyi, this doesn't work due to the import so we may want to remove it.

Reply:
@bkjohnson for this, should the language switch buttons just get added to the default template?

Reply:
I don't think we should have an internationalization story for the react component because we can't have that _and_ the prop documentation. If you can get the translation to work while also keeping the prop documentation that would be great, but if we have to choose the team decided that it's better to keep the documentation.

Reply:
Works for me, I'll just remove it. Thanks!

Reply:
just noticed the you already merged. Thanks again for all the help @bkjohnson